src/home_robot_hw/home_robot_hw/ros/recorder.py

```diff
@@ -21,11 +22,7 @@ class Recorder(object):
     def __init__(self, filename, start_recording=False, model=None, robot=None):
         """Collect information"""
         print("Connecting to robot environment...")
-        self.robot = (
-            HelloStretchROSInterface(visualize_planner=False, model=model)
-            if robot is None
-            else robot
-        )
+        self.robot = StretchManipulationEnv(init_cameras=True)
```

Comment:
should this be the new client by @exhaustin

Reply:
I think so...but I do not want to handle that change in this PR. I can make that change in my SLAP PR.

Reply:
file an issue